src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter(
+      person =>
+        Math.ceil(person.died / 100) === century
+        && person.sex === 'm'
+    )
+    : people.filter(person => person.sex === 'm');
```

Comment:
```suggestion
  const men = people.filter(
      person => century
        ? Math.ceil(person.died / 100) === century && person.sex === 'm'
        : person.sex === 'm'
      );
```

Reply:
no need to write filter method every time, just move ternary operator inside this method